static/src/javascripts/projects/common/modules/save-for-later.js

```diff
@@ -82,6 +84,14 @@ define([
         return { prop74: prefix + 'ContainerSave:' + contentId };
     };
 
+    SaveForLater.prototype.conditionalInit = function(){
+        if (!liveBlog.notificationsCondition()) {
+            var $saveForLaterContainer = $('.meta__save-for-later');
+            $saveForLaterContainer.show();
```

Comment:
We would normally want to keep styles in the CSS so instead of using the `show` method, use `addClass` and add a `meta__save-for-later--is-visible` class to the element.